src/status_im/ui/components/list_item/views.cljs

```diff
@@ -245,114 +247,144 @@
-;; makes sense since `title` is the key element of a list item
-
-;; title-row-accessory
-;; component - especially made for chat list item, but may serve other
-;; purpose in the unlikely future. Wrapper already has 2px :margin-top
-;; best to keep it <= 18px high
-
-;; subtitle
-;; any one of string, keyword representing translated string in the form of
-;; :t/{translation-key-in-translation-files},
-;; keyword(gets converted to string),
-;; number(gets converted to string), or
-;; component - when component is used best to keep the style similar
-;; to `styles/subtitle-title-row-container` and/or `styles/subtitle`
-
-;; subtitle-max-lines
-;; integer - 1 by default - self-explanatory
-
-;; subtitle-row-accessory
-;; component
-;; made specially for chat-list to hold unread messages counter
-
-;; content
-;; component - to replace entire title-column area
-;; TODO - consider removing, as it is redundant now that
-;; title/subtitle elements can be component as well
-;; just make sure to keep in mind the note made on
-;; component case on those keys above
-
-;; accessories
-;; vector of :chevron, :check, :more, string, number, keyword or component
-
-;; on-press/on-long-press
-;; function - self explanatory
-
-;; error
-;; string - error tooltip
-
-;; accessibility-label
-;; :keyword - self explanatory
-
-;; disabled?
-;; boolean - false by default - self explanatory
-
 (defn list-item
-  [{:keys [type theme container-margin-top container-margin-bottom
-           icon title-prefix title-prefix-width title-prefix-height
-           title title-color-override title-row-accessory
-           title-accessibility-label subtitle subtitle-max-lines
-           subtitle-row-accessory content accessories on-press
-           on-long-press error accessibility-label disabled?]
-    :or   {type                    :default
-           theme                   :default
-           disabled?               false
-           container-margin-top    0
-           container-margin-bottom 0
-           subtitle-max-lines      1}}]
+  "A general purpose Status-React specfic list item component.
+  Every key is optional. Use as needed.
+  Combination of around 4 related keys are enough for most cases.
+  Spec: https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=787%3A1108
+
+  `react-key`
+  String - (default generated automatically using `gensym`)
+  A react unique key meta data. Usually for homogeneous list-items.
+  Usually needed for list-items generated by looping over
+  some kind of collection.
+  More here https://reactjs.org/docs/lists-and-keys.html#keys
+
+  `type`
+  `:default` (default), `:small`, `:section-header`, or `:divider`
+  `:section-header`
+  Specifying only these is sufficient.
+  {:title <Section title> :type :section-header}
+  Optionally set `container-margin-top/bottom`
+  `:divider`
+  A simple common gray divider seen in various screens.
+  Simply use:
+  {:type :divider} and specify nothing else.
```

Comment:
btw it would be useful to have something like `:top-devider? bool` and `:bottom-devider? bool` options, to render divider on top or bottom of list-item

Reply:
I am not sure. That sounds like more of a concern/prop for `flat-list`, `scroll-view`, or a plain `view` which wrap/contain list-items. In case of a `flat-list`, it has `header` and `footer` that could contain a divider.

Furthermore, I was thinking `divider` to serve more as section divider(keeping in mind that some sections in some screen spec does not have a `divider` but only white space. cc @andmironov @errorists 

If we add them, they could serve(maybe abused in semantical sense) as what a `border-top` or `border-bottom` on list-item would serve. So far I only see `transaction popup` implementation needing those. Everywhere else, list-item does not have a border. I figured not having border in list-item spec was intentional. cc @errorists

Reply:
Yep, I guess the transaction overview modal seems to be the only place at the moment with list items separated by dividers. Due to high information density, dividers are quite necessary there, but it's more of an exception and in the rest of the app we tend not to use dividers, or as @bitsikka said, use them as a way to separate sections (e.g in settings)